packages/python/examples/flask/app.py

```diff
@@ -28,6 +28,7 @@ def grouping_function(request):
     MetricsApiConfig(
         api_key=os.getenv("README_API_KEY"),
         grouping_function=grouping_function,
+        background_worker_mode=False,
```

Comment:
Is this necessary?

Reply:
Yeah if we don't do this python will send it into a background queue (purged after one request) and we don't get the request in time.

Reply:
I'm not on mobile anymore so I can elaborate on this but I *think* what was going on was because `background_worker_mode` sends the request into a separate thread to make the request to metrics that was happening around the same time that we'd kill the HTTP server process. And because I implemented that group killing pid work to fix that issue with PHP that was killing the background process, resulting in no logs getting sent.